src/components/Modal.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import './Modal.css';
+
+export default class Modal extends Component {
```

Comment:
We don't have enough time to make this Modal component extensible, so in the meantime, we should name this component `HelpModal`. As such, all classes and file names should be `.HelpModal`-related.

Reply:
Done